libkbfs/kbfs_ops_concur_test.go

```diff
@@ -1315,7 +1327,7 @@ func TestKBFSOpsConcurWriteParallelBlocksError(t *testing.T) {
 	// leave ourselves in a dirty state.
 }
 
-// Test that writes that happen on a multi-block file concurrently
+// Test that writes that happens on a multi-block file concurrently
 // with a sync, which has to retry due to an archived block, works
```

Comment:
"When writes happen on a multi-block file concurrently with a sync, and the sync has to retry due to an archived block, test that everything works correctly."